### PR TITLE
542365460 Blank Resource group when using ARM task for specific subscription

### PR DESCRIPTION
**Pull request by PhilipsonJoseph**
**Task name**: AzureResourceManagerTemplateDeploymentV3

**Description**: Customer not able to view the resource group name as he have large no of RG in subscription. So added a logic to cumulate the response by pagination 

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) Y

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [X] Checked that applied changes work as expected

#https://portal.microsofticm.com/imp/v5/incidents/details/542365460/summary

### CODE REVIEW

**Comment by z00sts** [comment]
@PhilipsonJoseph can you point me logic changes in code? I can't see it, just updating the versions.

**Comment by PhilipsonJoseph** [comment]
> @PhilipsonJoseph can you point me logic changes in code? I can't see it, just updating the versions.

Hi @z00sts Changes is not much. Changes only on configuration which present on task.json

File: Tasks/AzureResourceManagerTemplateDeploymentV3/task.json [hunk 1]
```diff
@@ -246,10 +246,11 @@
...
       "endpointId": "$(ConnectedServiceName)",
-      "dataSourceName": "AzureResourceGroupsByExplicitSubscription",
-      "parameters": {
-        "subscriptionId": "$(subscriptionName)"
-      }
+      "endpointUrl": "{{{endpoint.url}}}/subscriptions/{{{endpoint.subscriptionId}}}/resourcegroups?$skiptoken={{{skiptoken}}}&api-version=2016-02-01",
+      "callbackContextTemplate": "{\"skiptoken\": \"{{#getTokenValue response.nextLink}}{{extractUrlQueryParameter %24skiptoken}}{{/getTokenValue}}\"}",
+      "callbackRequiredTemplate": "{{isTokenPresent response.nextLink}}",
+      "initialContextTemplate": "{\"skiptoken\": \"\"}",
+      "resultSelector": "jsonpath:$.value[*].name"
```
Comment (z00sts) [inline Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:253]: Do we have other tasks with the same logic we should also cover with this change?

Comment (PhilipsonJoseph) [reply Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:253]: In none of the task we have a pagination logic implemented for Resource group but we received ICM for this task and we believe it is due to pagination, Based on the customer response we may take it as repair item to implement for other task

**Pull request merged**